src/source.F90

```diff
@@ -162,8 +156,21 @@ subroutine sample_external_source(site)
           end if
         end if
       end select
+
+      ! Check for rejection
+      if (.not. found) then
+        n_reject = n_reject + 1
+        if (n_reject >= EXTSRC_REJECT_THRESHOLD .and. &
+             real(n_sample, 8)/n_reject <= EXTSRC_REJECT_FRACTION) then
```

Comment:
Perhaps I'm confused, but won't `real(n_sample, 8) /n_reject` always be greater than unity? I *think* that `n_sample` will always be greater than or equal to `n_reject`, but I may be wrong. But if so, then I think the quotient and dividend should be interchanged to compute the rejection fraction.